src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
Yes, but that doesn't mean we have to specifically target broken tools that are incompatible with the majority of C compilers (with the notable exception of Sun/Oracle Solaris Studio C/C++).

But to make you happy, I found a better solution than the proposed `sed` script while retaining compatibility with the Solaris tools. See https://github.com/mmaker/ocaml-nocrypto/commit/6d17130f92991059b02b0c69e7fab9e9e221d393